### ISSUE TITLE
Consistent selection highlight colour

### ISSUE THREAD

**Issue by gilbertohasnofb** [marktext/marktext#2372]
It would be great to have a consistent selection highlight colour between Source Code Mode and Regular Mode. Using the Dark theme, it is very difficult to see a selection in Source Code Mode because it is so dark that it nearly matches the background, but on the other hand it is very easy to see the selection in Regular Mode as it is substantially brighter. 

This lack of consistence is also present with other themes, though Dark is particularly problematic due to the lack of contrast between selection and background in Source Code Mode.

**Comment by brainchild0** [comment]
I think that source code mode is currently implemented in an ad-hoc way. Hopefully   a cleaner implementation would be possible at some point, at which time better attention to cosmetic details would be a sensible consideration.

**Comment by gilbertohasnofb** [comment]
It is not only about consistency, though. In the case of the Dark theme, you basically cannot select any text since you cannot see where is your selection, so this is definitely not just a cosmetic issue. See example with all three dark themes below:

Dark (basically impossible to see):

![image](https://user-images.githubusercontent.com/5700795/97801723-98a9ab00-1c36-11eb-8c82-f530eb33a13e.png)

Material Dark (difficult to see but somewhat visible):

![image](https://user-images.githubusercontent.com/5700795/97801713-86c80800-1c36-11eb-89a7-92143a1a94c8.png)

Material Dark (super easy to see):

![image](https://user-images.githubusercontent.com/5700795/97801733-b0812f00-1c36-11eb-9d13-aa5f1acbd8e5.png)

All white themes are very easy to see too.

**Comment by bassmanac** [comment]
I agree, both the Dark and Material Dark themes are next to impossible to see selections in source code mode. They should match the color of selections in the regular mode which are much easier to see.

👍
